src/main/java/com/uwetrottmann/tmdb2/Tmdb.java

```diff
@@ -52,6 +98,61 @@ public Tmdb(String apiKey) {
         this.apiKey = apiKey;
     }
 
+    /**
+     * Create a new authenticated manager instance for post/delete operations.
+     *
+     * @param apiKey Your TMDB API key.
+     */
+    public Tmdb(String apiKey, String username, String password) throws IOException, TmdbException {
+        this.apiKey = apiKey;
+        authenticate(username, password);
+    }
+
+
+    public void authenticateAnonymously() throws IOException, TmdbServiceErrorException {
+        AuthenticationService authService = getRetrofit().create(AuthenticationService.class);
+        GuestSession session = authService.createGuestSession().execute().body();
```

Comment:
[As mentioned](https://github.com/UweTrottmann/tmdb-java/pull/44#discussion_r118417242) this should be handled by an [authenticator](https://github.com/square/okhttp/wiki/Recipes#handling-authentication). Here one should only pass if wanting to authenticate anonymously or provide credentials.

The interceptor should add auth if it has it, but not through if it does not. The authenticator should try to auth and throw (or just return null) if it can't.